src/models/userModel.ts

```diff
@@ -0,0 +1,17 @@
+import { model, Schema } from 'mongoose'
+
+interface IUser {
+  name: string
+  email: string
+  password: string
+}
```

Comment:
Adicionar as propriedades: 
```suggestion
interface IUser {
  name: string
  email: string
  password: string
  document: string
  image: string
  address: string
  city: string
  state: string
  code: number
  complement: string
}
```